packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -38,69 +44,91 @@ export const AuthModal = ({
     onLoginSuccess = noop,
     onRegistrationSuccess = noop,
     onPasswordResetSuccess = noop,
+    isOpen,
+    onOpen,
+    onClose,
     ...props
 }) => {
     const {formatMessage} = useIntl()
-    const customer = useCustomer()
+    const customerId = useCustomerId()
```

Comment:
I agree with you on the scope of the `useCustomerId` hook is confusing when we have `useCustomer` that behavior very differently but have similar names.

But i don't have a very good alternative for this yet, i don't hate `useCurrentCustomerId` but also don't love it.

We are going to have a similar issue for basketId, i'd like to finish implementing the basket and come back to this issue, I'm hoping that we'll have more clear understanding of the issue with these 2 instances and can come up with  a better naming/pattern. 

TL;DR can we fix this later?

Reply:
Yeah.. no action required there. I think this would become part of that global context we talked about one time before. You can resolve this conversation.